client/settings/fraud-protection/advanced-settings/cards/order-items-threshold.tsx

```diff
@@ -48,40 +40,29 @@ const OrderItemsThresholdCustomForm: React.FC< OrderItemsThresholdCustomFormProp
 		[ protectionSettingsUI, setting ]
 	);
 
-	const minItemsTemp = parseInt( settingUI.min_items + '', 10 );
-	const maxItemsTemp = parseInt( settingUI.max_items + '', 10 );
+	const minItems = parseInt( settingUI?.min_items + '', 10 );
+	const maxItems = parseInt( settingUI?.max_items + '', 10 );
```

Comment:
Yes, I think it would be better to use `Number` here instead of the multiple `parseInt`, but I'd be more comfortable having it in a different PR. I can open an enhancement issue to address that. It could even be something that we do on cooldown weeks. What do you think?

Reply:
Yep, definitely something to explore on a cooldown or similar 🙂